.github/actions/verify-token/dist/dist/index.js

```diff
@@ -82,7 +82,7 @@ function run() {
                 GITHUB_REF_TYPE="branch" \
                 GITHUB_ACTOR="laurentsimon" \
                 GITHUB_WORKSPACE="$(pwd)" \
-                nodejs ./dist/index.js
+                nodejs ./dist/dist/index.js
```

Comment:
why is this sub-nested now?

Reply:
This is a temporary fix for https://github.com/slsa-framework/slsa-github-generator/issues/1668 that @ianlewis sent.

Reply:
ah, thank you!